js/datastore.md

```diff
@@ -60,7 +60,7 @@ npm i -g @aws-amplify/cli@latest
 The Amplify CLI can generate models at any time with the following command:
 
 ```
-npm run amplify-modelgen
+amplify codegen model
```

Comment:
This doesn't need to change. `npm run amplify-modelgen` should work.

Reply:
`npm run amplify-modelgen` only works if you generate with NPX. The text in above the command talks about the Amplify CLI, which should be the `amplify codegen model` version.